config.yaml

```
@@ -57,20 +57,28 @@ params:
   about:
     enable: true
     title: About Us
-    subtitle: This is the community of Python programmers in Pune and beyond.
+    subtitle: This is the community of **Python programmers** and **Free and Open-source (FOSS) lovers** in Pune and beyond.
     row:
     - list:
       - icon: fa-users
         title: Community
-        description: ''
+        description: 'We are a diverse community of attendees and
```

```suggestion
        description: |
          We are a diverse community of attendees and…
```

It will be better if we use this construct from YAML, that way we can add more text to the sections which can have new lines as well.
https://yaml.org/spec/1.2/spec.html#style/block/literal